### DISCUSSION TITLE
Release plan?

### DISCUSSION THREAD

**Discussion by ottapav** [qpsolvers/qpsolvers#271] [General]
Hi Stephane,
Is there any release plan? Maybe github milestones to see when the package will be released with what?

**Comment by stephane-caron** [comment]
Hi @ottapav, there as been no release plan so far. New minor releases happen when someone contributes a PR, e.g. adding a new solver interface, which typically happens not-so-long after a new solver with Python bindings is released. Major releases are [announced](https://github.com/qpsolvers/qpsolvers/discussions/categories/announcements): I did it properly [for v3](https://github.com/qpsolvers/qpsolvers/discussions/147), but not for v4 where the breaking change was restricted to a single function with few users (still, it should have been done).

Feel free to detail if you have specific needs that a release plan would address, and that are not currently addressed by subscribing to project announcements.

**Reply by ottapav** [reply]
Hi @stephane-caron, I'm just curious when the new Qpsolvers version will be released. If there is a plan to have some features in the new release, it would be nice to use the milestone view https://github.com/qpsolvers/qpsolvers/milestones?state=open - it gives the feeling that something new is coming. However, I understand releasing on demand.

**Reply by ottapav** [reply]
@stephane-caron When it is possible to release the current changes? It would be handy to me, if it is rather sooner :).

**Reply by stephane-caron** [reply]
Sure thing, I will release 4.3.0 today or tomorrow: https://github.com/qpsolvers/qpsolvers/pull/280.
